--- a/arbitration/Arbitration.tla
+++ b/arbitration/Arbitration.tla
@@ -432,7 +432,11 @@
   /\ ConfigAcceptable(f, config[A], proposal[A])
   /\ proposal'      = [proposal EXCEPT ![f] = proposal[A]]
   /\ config'        = [config EXCEPT ![f] = config[A]]
-  /\ log'           = [log EXCEPT ![f] = TruncateStaleEntries(f, config[A])]
+  /\ log'           = [log EXCEPT ![f] =
+                         IF Len(log[f]) > config[A].barrier.index
+                            /\ log[f][config[A].barrier.index + 1] < proposal[A]
+                         THEN SubSeq(log[f], 1, config[A].barrier.index)
+                         ELSE log[f]]
   /\ role'          = RoleAfter(f, proposal[A])
   /\ promisedBy'    = PromisedByAfter(f, proposal[A])
   /\ prepareConfig' = PrepareConfigAfter(f, proposal[A])
