--- a/arbitration/Arbitration.tla
+++ b/arbitration/Arbitration.tla
@@ -349,7 +349,11 @@
   /\ CanReceiveMeta(f, meta[A], pid[A])
   /\ pid'      = [pid EXCEPT ![f] = pid[A]]
   /\ meta'     = [meta EXCEPT ![f] = meta[A]]
-  /\ log'      = [log EXCEPT ![f] = LogAfterMeta(f, meta[A])]
+  /\ log'      = [log EXCEPT ![f] =
+                   IF Len(log[f]) > meta[A].barrier.idx
+                      /\ log[f][meta[A].barrier.idx + 1] < pid[A]
+                   THEN SubSeq(log[f], 1, meta[A].barrier.idx)
+                   ELSE log[f]]
   /\ role'     = RoleAfter(f, pid[A])
   /\ promises' = PromisesAfter(f, pid[A])
   /\ pq'       = PqAfter(f, pid[A])
