--- a/arbitration/Arbitration.tla
+++ b/arbitration/Arbitration.tla
@@ -578,7 +578,6 @@
   \/ \E l \in F, v \in Server         : AcceptConfig(v, l)
   \/ \E l \in F                       : CommitConfig(l)
   \/ \E f \in F                       : ArbiterPushConfig(f)
-  \/ \E f \in F                       : ArbiterAdoptProposal(f)
   \/ \E l \in F                       : ClientWrite(l)
   \/ \E f, l \in F                    : AcceptEntry(f, l)
   \/ \E l \in F, i \in 1..MaxLogLen   : CommitEntries(l, i)
@@ -785,7 +784,6 @@
   \/ \E l \in F, v \in Server       : AcceptConfig(v, l)
   \/ \E l \in F                     : CommitConfig(l)
   \/ \E f \in F                     : ArbiterPushLive(f)
-  \/ \E f \in F                     : ArbiterAdoptLive(f)
   \/ \E l \in F                     : ClientWrite(l)
   \/ \E f, l \in F                  : AcceptEntry(f, l)
   \/ \E l \in F, i \in 1..MaxLogLen : CommitEntries(l, i)
@@ -802,7 +800,6 @@
   /\ \A l \in F, v \in Server       : WF_vars(AcceptConfig(v, l))
   /\ \A l \in F                     : WF_vars(CommitConfig(l))
   /\ \A f \in F                     : WF_vars(ArbiterPushLive(f))
-  /\ \A f \in F                     : WF_vars(ArbiterAdoptLive(f))
   /\ \A f, l \in F                  : WF_vars(AcceptEntry(f, l))
   /\ \A l \in F, i \in 1..MaxLogLen : WF_vars(CommitEntries(l, i))
 
